libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/SearchBar.kt

```diff
@@ -184,10 +184,10 @@ object ElementSearchBarDefaults {
 @Immutable
 sealed interface SearchBarResultState<in T> {
     /** No search results are available yet (e.g. because the user hasn't entered a search term). */
-    class NotSearching<T> : SearchBarResultState<T>
+    class Empty<T> : SearchBarResultState<T>
```

Comment:
I feel like `NotSearching` was more meaningful than `Empty`, if I saw this class for the first time I'd have some trouble differentiating between `Empty `and `NoResultsFound`, to be honest.

Maybe we could call it `Idle` or `Initial`?

Reply:
Yeah but NotSearching wasn't true, it's just an Empty state where you show nothing. But Initial was also on my mind, so I can change if u prefer.